k8s-readonly.tf

```diff
@@ -29,6 +29,9 @@ resource "kubernetes_deployment" "apiary_hms_readonly" {
           name = "${local.hms_alias}-readonly"
         }
         annotations = {
+          "ad.datadoghq.com/hms-readonly.check_names": "[\"prometheus\"]"
+          "ad.datadoghq.com/hms-readonly.init_configs": "[{}]"
+          "ad.datadoghq.com/hms-readonly.instances":"[{ \"prometheus_url\": \"http://%%host%%:8080/actuator/prometheus\", \"namespace\": \"hms-readonly\", \"metrics\": [ \"metrics_init_total_count_tables_value\", \"metrics_init_total_count_dbs_value\", \"metrics_memory_heap_used_value\", \"metrics_init_total_count_partitions_value\", \"metrics_memory_heap_max_value\", \"metrics_threads_count_value\", \"metrics_classloading_loaded_value\"], \"type_overrides\": { \"metrics_init_total_count_dbs_value\": \"gauge\", \"metrics_init_total_count_tables_value\": \"gauge\", \"metrics_memory_heap_used_value\": \"gauge\", \"metrics_init_total_count_partitions_value\": \"gauge\", \"metrics_memory_heap_max_value\": \"gauge\", \"metrics_threads_count_value\": \"gauge\", \"metrics_classloading_loaded_value\": \"gauge\" }, \"send_histograms_buckets\": true, \"send_monotonic_counter\": true, \"send_distribution_buckets\": true, \"send_distribution_counts_as_monotonic\": true }]"
```

Comment:
Assuming the metrics in both R/W and R/O are the same, wouldn't it be better to put the metric names in the variables file and use them in both tf files?

Reply:
Yes, that makes sense, I will put it in a variable. Thanks for the review.

Reply:
I have moved them to the variables file.